### ISSUE TITLE
sometimes receives 500 messages when asked for last one forex get

### ISSUE THREAD

**Issue by gchumbes** [finnhubio/Finnhub-API#64]
hi when i test this one
https://finnhub.io/api/v1/forex/candle?symbol=BINANCE:LTCBTC&resolution=1&count=1&token=bpuac87rh5r9b3jgg7fg

sometimes receive one message as expected some others 500 (last 500 as i realize)

why?

**Comment by finnhubio** [comment]
I do not understand your questions. Can you rephrase it a bit?

**Comment by gchumbes** [comment]
hi when i made this request:https://finnhub.io/api/v1/forex/candle?symbol=BINANCE:LTCBTC&resolution=1&count=1&token=bpuac87rh5r9b3jgg7fg

since resolution = 1 and count = 1 i expected to get last candle, which i get (usually) but yesterday sometimes i got last and unique message but with the same request sometimes i got the 500 last candles of that crypto, i used that link and the following:
https://finnhub.io/api/v1/crypto/candle?symbol=BINANCE:LTCBTC&resolution=1&count=1&token=bpuac87rh5r9b3jgg7fg same result, i just tried again same problem, should that work in that way? why?
